README.md

```diff
@@ -34,6 +34,10 @@ These are the main priorities for the near future:
 - The User Interface is basically still Bisq. Needs to be completely reworked and adapted for Monero as base currency. The new design is discussed and developed in [haveno-design](https://github.com/haveno-dex/haveno-design)
 - Cleanup the repository from Bisq-specific content (https://github.com/haveno-dex/haveno/projects/1)
 
+### Bounties
+
+To incentivize development we adopt a simple bounty system. Contributors may be awarded bounties after they completed a task (resolved and issue). [More details in the docs](https://github.com/erciccione/haveno/blob/master/docs/bounties.md).
```

Comment:
Suggestions:

"Contributors may be awarded bounties after completing a task (resolving an issue)."
or
"Contributors may be awarded bounties after resolving an issue with an associated bounty."